src/login/redux/sagas/login/postUpdatedTouAcceptSaga.js

```diff
@@ -0,0 +1,39 @@
+import { call, select, put } from "redux-saga/effects";
+import postRequest from "../postDataRequest";
+import { putCsrfToken } from "../../../../sagas/common";
+import * as actions from "../../actions/postUpdatedTouAcceptActions";
+import { nextMockUrlMfa } from "../../actions/postRefLoginActions";
+import {
+  loadingData,
+  loadingDataComplete,
+} from "../../actions/loadingDataActions";
+import { eduidRMAllNotify } from "../../../../actions/Notifications";
+
+export function* postUpdatedTouAcceptSaga(action) {
+  const state = yield select((state) => state);
+  const url = "https://idp.eduid.docker/tou";
+  try {
+    const dataToSend = {
+      ref: state.login.ref,
+      csrf_token: state.config.csrf_token,
+      user_accepts: action.payload.user_accepts,
+    };
+    yield put(eduidRMAllNotify());
```

Comment:
I just wonder why we are removing notification here? since it would be a empty message anyway